### ISSUE TITLE
Provide to the team a generic script they could use to SSH to a VM

### ISSUE THREAD

**Issue by jacobdotcosta** [snowdrop/team#751]

**Comment by cmoulliard** [comment]
Is it possible that you move the highlighted text to a comment, otherwise it is displayed by the new reporting tool ? @jacobdotcosta

**Comment by jacobdotcosta** [comment]
e.g.

```bash
#!/usr/bin/env bash

#set -x

VM_NAME=$1
SSH_KEY=~/.ssh/id_rsa_snowdrop_hetzner_${VM_NAME}

# Copy the private key
echo "$(PASSWORD_STORE_DIR=~/.password-store-snowdrop pass show hetzner/${VM_NAME}/id_rsa)" > ~/.ssh/id_rsa_snowdrop_hetzner_${VM_NAME}
chmod 600 ~/.ssh/id_rsa_snowdrop_hetzner_${VM_NAME}

# Extract the IP, PORT and USER to be used to ssh
IP=$(PASSWORD_STORE_DIR=~/.password-store-snowdrop pass show hetzner/${VM_NAME}/ansible_ssh_host | awk 'NR==1{print $1}')
PORT=$(PASSWORD_STORE_DIR=~/.password-store-snowdrop pass show hetzner/${VM_NAME}/ansible_ssh_port | awk 'NR==1{print $1}')
USER=$(PASSWORD_STORE_DIR=~/.password-store-snowdrop pass show hetzner/${VM_NAME}/os_user | awk 'NR==1{print $1}')

echo "### SSH COMMAND: ssh -i $SSH_KEY $USER@$IP -p $PORT"
ssh -i ${SSH_KEY} ${USER}@${IP} -p ${PORT} "${@:2}"
```

**Comment by jacobdotcosta** [comment]
Implemented on snowdrop/k8s-infra#240
